client/src/store/state.js

```diff
@@ -1,6 +1,11 @@
 export default {
   id: 2,
-  isLogged: false,
+  isLogged: true,
   username: 'Lakshya Khera',
-  position: null
+  position: null,
+  SettingState: {
+    isPActive: true,
+    isIActive: false
+  },
+  newsAPI_KEY: '9ad299f940c94a938272d3c3b5d94363'
```

Comment:
is this the best way to store keys?

Reply:
I've added the keys in the state right now, though later we can fetch keys from backend when client uses the news widget, is it possible @capedcrusader23 ?

Reply:
I think you should have stored them in a secret.js file and require that file inside your vue file and gitignore that file. So others can put their key in that while using in their local instead they use your API key.

Reply:
This is for the test, I'll change it asap. :D Thanks for advice.

Reply:
You are welcome.

Reply:
I think @TG1999  is right store in secret.js and gitignore them and yes @lakshyabatman it is possible you have to create a function which on mount will fetch the keys from api from backend.

Reply:
Fixed in the next commit :dagger: